nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -139,6 +139,8 @@
 
 - [c2FmZQ](https://github.com/c2FmZQ/c2FmZQ/), an application that can securely encrypt, store, and share files, including but not limited to pictures and videos. Available as [services.c2fmzq-server](#opt-services.c2fmzq-server.enable).
 
+- [forgejo](https://forgejo.org/), a git forge. Previously deployed as a drop-in to gitea. Available as [services.forgejo](#opt-services.forgejo.enable). See migration instructions in [manual](https://nixos.org/manual/nixos/stable#module-forgejo)
```

Comment:
```suggestion
- [forgejo](https://forgejo.org/), a git forge. Previously deployed as a drop-in replacement package in the gitea module. Available as [services.forgejo](#opt-services.forgejo.enable). See migration instructions in [manual](https://nixos.org/manual/nixos/stable#module-forgejo)
```

Reply:
Possibly also add that the migration from the drop-in package with gitea module to the new forgejo module does not have to happen immediately. This can be relevant for automated rollouts of unattended upgrades, where manual migration steps shall be avoided.

Reply:
:heavy_check_mark: